x11-misc/gmrun/gmrun-1.4w.ebuild

```diff
@@ -0,0 +1,31 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+if [[ ${PV} == 9999 ]]; then
+	inherit git-r3
+	EGIT_REPO_URI="https://github.com/WdesktopX/${PN}.git"
+	EGIT_BRANCH="master"
+else
+	SRC_URI="https://github.com/WdesktopX/${PN}/releases/download/${PV}/${P}.tar.xz"
+	KEYWORDS="~amd64 ~mips ~ppc x86"
+fi
+
+DESCRIPTION="A GTK-2 based launcher box with bash style auto completion!"
+HOMEPAGE="https://github.com/WdesktopX/gmrun/"
+
+LICENSE="GPL-1"
```

Comment:
this seems incorrect on that new fork, seems like the author invented his/her own license

Reply:
it is "ISC" 

https://github.com/WdesktopX/gmrun/commit/ed2760ac9745c7f7b3aecf850a590e683786a57a